node-backend/app/controllers/v1/usersController.js

```diff
@@ -1,33 +1,92 @@
+const qs = require(/*eslint-disable  node/no-extraneous-require*/ "qs");
+
 const db = require("../../models/sequelize");
 const Users = db.users;
-
 const jwtToken = require("../../jwtTokenValidation/jwtValidation");
 module.exports.findUsersByOrg = async (req, res) => {
   const authHeader = req.headers["authorization"];
   let userData = await jwtToken.getData(authHeader);
-  Users.findAll({
-    where: { org_id: userData.orgId, role_id: 2 },
-    attributes: [
-      "id",
-      "first_name",
-      "email",
-      "display_name",
-      "profile_image_url",
-      "role_id",
-      "hi5_quota_balance",
-      "org_id",
-    ],
-  })
-    .then((users) => {
-      res.status(200).send({
-        data: users,
-      });
+  let limit;
+  let offset;
+  let obj = qs.parse(req.query);
+  if (obj.limit) {
+    limit = obj.limit;
+  } else {
+    limit = 10;
+  }
+  if (obj.offset) {
+    offset = obj.offset;
+  } else {
+    offset = 0;
+  }
+  if (obj.org_id) {
+    let adminRoleId = await Users.findOne({
```

Comment:
you will be receiving role_id from token itself. Why you need to check from database?